docs/3.Custom Policies/Examples.md

```diff
@@ -421,3 +421,17 @@ definition:
     operator: "contains"
     value: "0.0.0.0/0"
 ```
+
+## Prevent the use of an unauthorized module source
+
+The following policy will pass if and only if the source begins with `terraform-aws-modules`.
+
+```yaml
+definition:
+  cond_type: attribute
+  resource_types:
+    - module
+  attribute: source
+  operator: starting_with
+  value: terraform-aws-modules
+```
```

Comment:
a module policy example exists already at line 27 🙂

Reply:
Search failed me because the file was too long 🤦